Source/Particles/Collision/MCCScattering.H

```diff
@@ -195,6 +196,8 @@ public:
     *
     * @param[in] energy_cost energy cost of ionization
     * @param[in] mass1 mass of the colliding species
+    * @param[in] ion_vel_std standard deviation of ion velocity distribution; for
+    *                        a Maxwellian distribution it should equal sqrt(kB*T/m)
```

Comment:
Out of curiosity, is there maybe some factor like `3/2` that should multiply `kB*T/m` in 3D vs. 2D, when we say that the velocity should equal `sqrt(kB*T/m)`?

Reply:
Seeing as all WarpX simulations use 3 velocity components this won't change depending on the simulation dimension. This is the standard deviation of the normal distribution used to sample each velocity component.

Reply:
Sounds legit. Or we write proportional here.

cc @roelof-groenewald :)

Reply:
@ax3l How about we change the docstring to be:
`standard deviation of ion velocity distribution for each direction; for a Maxwellian distribution it should equal sqrt(kB*T/m)`.
If we wanted we could update that function to take separate arguments for the "temperature" in `vx`, `vy` and `vz`, but for now we are assuming an isotropic distribution.

Reply:
That works for me; @EZoni ? :)

Reply:
The new docstring sounds good to me! Thank you both for giving feedback on my question!